docs/configuration.md

```diff
@@ -456,6 +456,29 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
+  <td><code>spark.sql.repl.eagerEval.enabled</code></td>
+  <td>false</td>
+  <td>
+    Enable eager evaluation or not. If true and repl you're using supports eager evaluation,
+    dataframe will be ran automatically and html table will feedback the queries user have defined
+    (see <a href="https://issues.apache.org/jira/browse/SPARK-24215">SPARK-24215</a> for more details).
+  </td>
+</tr>
+<tr>
+  <td><code>spark.sql.repl.eagerEval.showRows</code></td>
```

Comment:
I don't think it is obvious what "showRows" does. I would assume that it is a boolean, but it is a limit instead. What about calling this "limit" or "numRows"?

Reply:
maxNumRows

Reply:
Thanks, change it in 94f3414.